sklearn/decomposition/_kernel_pca.py

```diff
@@ -358,10 +359,11 @@ def inverse_transform(self, X):
         ----------
         "Learning to Find Pre-Images", G BakIr et al, 2004.
         """
-        if not self.fit_inverse_transform:
-            raise NotFittedError("The fit_inverse_transform parameter was not"
-                                 " set to True when instantiating and hence "
-                                 "the inverse transform is not available.")
+        if not self.enable_inverse_transform:
+            raise NotFittedError("The enable_inverse_transform parameter was"
```

Comment:
As the codecov bot discovered, the existing tests for not trigger this branch of the code. Could you please add a quick test for this (using `pytest.raises(NotFittedError)`)?